pkg/fb/api.go

```diff
@@ -0,0 +1,371 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package fb
+
+import (
+	"time"
+
+	flatbuffers "github.com/google/flatbuffers/go"
+
+	v1 "github.com/apache/skywalking-banyandb/api/fbs/v1"
+	"github.com/apache/skywalking-banyandb/pkg/convert"
+)
+
+type ComponentBuilderFunc func(*flatbuffers.Builder)
+
+type WriteEntityBuilder struct {
+	*flatbuffers.Builder
+}
+
+func NewEntityBuilder() *WriteEntityBuilder {
```

Comment:
Please insert `Write` to the func name since both `write` and `query` builders are placed in this file.